front/deployfile_download.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ * GLPI Inventory Plugin
+ * Copyright (C) 2021 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on FusionInventory for GLPI
+ * Copyright (C) 2010-2021 by the FusionInventory Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI Inventory Plugin.
+ *
+ * GLPI Inventory Plugin is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI Inventory Plugin is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with GLPI Inventory Plugin. If not, see <https://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+include("../../../inc/includes.php");
+
+Session::checkRight('plugin_glpiinventory_package', READ);
+global $CFG_GLPI;
+
+$deployfile_id = null;
+if (isset($_GET['deployfile_id'])) {
+    $deployfile_id = urldecode($_GET['deployfile_id']);
+}
+
+$deploy = new PluginGlpiinventoryDeployFile();
+if ($deploy->getFromDB($deployfile_id)) {
```

Comment:
Param is expected to be an integer. No need to use `urldecode`.
```suggestion
$deployfile_id = (int)($_GET['deployfile_id'] ?? 0);

$deploy = new PluginGlpiinventoryDeployFile();
if ($deployfile_id > 0 && $deploy->getFromDB($deployfile_id)) {
```